content/hardware/07.opta/opta-family/opta/tutorials/02.getting-started-with-rs485/content.md

```diff
@@ -29,10 +29,10 @@ This tutorial will show the steps to connect two Opta™ devices via RS-485 and
 
 ### Required Hardware and Software
 
-- Opta™ PLC with RS-485 support (x2)
-- 12-24VDC/1A power supply (x1)
+- [Opta™ PLC with RS-485 support](https://store.arduino.cc/products/opta-wifi) (x2)
```

Comment:
I see we're linking to the WiFi variant, and I'm good with it. But as a suggestion, would it be better to redirect them to the RS485 variant? To show the RS485 name tag explicitly when redirected:

```suggestion
- [Opta™ PLC with RS-485 support](https://store.arduino.cc/products/opta-rs485) (x2)
```

or the following change to include both variants so they have options? With the idea of letting them know both variants are compatible as well. 

```suggestion
- Opta™ PLC with RS-485 support: [Opta™ RS485](https://store.arduino.cc/products/opta-rs485), or [Opta™ WiFi](https://store.arduino.cc/products/opta-wifi) (x2)
```